apps/browser/qml/pages/components/PopUpMenuFooter.qml

```diff
@@ -27,40 +27,40 @@ Rectangle {
         height: root.height
 
         Shared.IconButton {
-            id: forwardButton
+            id: closeTabButton
             width: Theme.itemSizeLarge
-            icon.source: "image://theme/icon-m-forward"
+            icon.source: "image://theme/icon-m-tab-close"
             icon.opacity: enabled ? 1.0 : Theme.opacityLow
-            enabled: webView.canGoForward
+            enabled: webView.tabModel.count > 0
             onTapped: {
-                webView.goForward()
-                overlay.animator.showChrome()
+                webView.tabModel.closeActiveTab()
+                if (webView.tabModel.count === 0) {
+                    overlay.startPage(PageStackAction.Animated)
+                }
             }
         }
 
         Shared.IconButton {
-            id: shareButton
+            id: backwardButton
             width: Theme.itemSizeLarge
-            icon.source: "image://theme/icon-m-share"
+            icon.source: "image://theme/icon-m-back"
             icon.opacity: enabled ? 1.0 : Theme.opacityLow
-            enabled: webView.contentItem
+            enabled: webView.canGoBack
```

Comment:
Back button should not be here. It's already there in the toolbar and visible if you can go back.

Reply:
So the idea behind that is, if you ever want to do advanced things with you tab, like close, go forward, etc, I often combine that with going backwards, reloading, etc (some pages are really annoying). So imo it is the most consistent to have all controls you need for that available in one menu section. Otherwise I need to remember to close the menu, when I hit forward one time too many and need to go back again. In my personal opinion the UX is best, if you have close, backward, forward and reload in one row.

Reply:
That advanced use is very much speculative for boarder audience. From UX point of this once again increases the menu width is not good as it reduces area that can be used for closing the menu.

Checked this with design we concluded following order for the bottom part of the menu:
"Close" / "Forward" /  "Favorite" / "Reload"

"Share" can be moved to be a menu item.

Reply:
Well, you can just swipe down the menu, but I can remove the back button, if it really is an issue. It does have the sideeffect of the menu being easier to use with your left thumb though.

Reply:
Yeap, it was raised please drop. If you're on top of this, could also rebase the PR. Thank you in advance!

Reply:
Done

Reply:
Thank you! Almost there. Please update also icon order to the menu:

"Close" / "Forward" / "Favorite" / "Reload"

Just swap "Favorite" and "Reload" items.

See: https://github.com/sailfishos/sailfish-browser/pull/905#discussion_r785692823

Reply:
Oops, I missed that, my bad!